packages/web3-core/types/tests/abstract-web3-module-test.ts

```diff
@@ -67,13 +68,22 @@ abstractWeb3Module.defaultGasPrice;
 abstractWeb3Module.defaultGas;
 
 // $ExpectType Providers
-abstractWeb3Module.providers
+abstractWeb3Module.providers;
```

Comment:
Only the static property will exist.

Reply:
Ok can you fix in this branch as not around on a computer today!